packages/ui-app/src/Params/types.d.ts

```diff
@@ -16,6 +17,7 @@ export type RawParam$Values = RawParam$Value | RawParam$ValueArray;
 
 export type RawParam = {
   isValid: boolean,
+  key: StorageFunction,
```

Comment:
This is not always the case, e.g. for Extrinsics, RawParam is used and it is not storage at all.